addons/print/views/print_view.xml

```diff
@@ -0,0 +1,210 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <!-- print.provider views -->
+        <record id="print_provider_form_view" model="ir.ui.view">
+            <field name="name">print.provider.form</field>
+            <field name="model">print.provider</field>
+            <field name="arch" type="xml">
+                <form string="Postal Provider">
+                    <sheet>
+                        <group name="provider_base" string="Provider Information">
+                            <field name="name"/>
+                            <field name="provider"/>
+                            <field name="environment"/>
+                            <label for="balance"/>
+                            <div>
+                                <field name="balance" readonly="1" class="oe_inline"/>
+                                <button name="update_account_data"
+                                        string="Update"
+                                        type="object"
+                                        colspan="1"/>
+                            </div>
+                        </group>
+                        <notebook>
+                        </notebook>
+                    </sheet>
+                </form>
+            </field>
+        </record>
+
+        <record id="print_provider_list_view" model="ir.ui.view">
+            <field name="name">print.provider.list</field>
+            <field name="model">print.provider</field>
+            <field name="arch" type="xml">
+                <tree string="Postal Provider">
+                    <field name="name"/>
+                    <field name="provider"/>
+                    <field name="environment"/>
+                </tree>
+            </field>
+        </record>
+
+        <record id="print_provider_search_view" model="ir.ui.view">
+            <field name="name">print.provider.search</field>
+            <field name="model">print.provider</field>
+            <field name="arch" type="xml">
+                <search>
+                    <filter name="environment_live" string="Provider Prod." domain="[('environment','=', 'prod')]"/>
+                    <filter name="environment_test" string="Provider Test" domain="[('environment','=', 'test')]"/>
```

Comment:
Are those filters really necessary ? I mean, will you use them in an every day production server ?